tests/tests.coffee

```diff
@@ -93,6 +93,49 @@ class VexTabTests
     tab.getArtist().render(renderer)
     assert.ok(true, "all pass")
 
+  makeCanvas = (vex, test_id, flex)->
+    c = $('<div></div>').css('flex', flex).css('font-size', '0.8em')
+    p = $('<p></p>').css('margin-top', '0px')
+    p.append($('<pre></pre>').text(vex).css('font-family', 'courier'))
+    c.append(p)
+    canvas = $('<div></div>').addClass("vex-tabdiv").attr('id', test_id).css('flex', flex)
+    c.append(canvas)
+    return c
+
+  renderCodeInCanvasId = (code, canvasid) ->
+    tab = new VexTab(new Artist(0, 0, 500, {scale: 0.8}))
+    tab.parse(code)
+    canvas = $('#' + canvasid)
+    renderer = new Vex.Flow.Renderer(canvas[0], Vex.Flow.Renderer.Backends.SVG)
+    renderer.getContext().setBackgroundFillStyle("#eed")
+    tab.getArtist().render(renderer)
+
+  # Render content to a new div, and return the content.
+  # Remove some things that change but aren't relevant (IDs)
+  getRenderedContent = (container, code, cssflex) ->
+    idcounter += 1
+    canvasid = 'rendered-' + idcounter
+    container.append(makeCanvas(code, canvasid, cssflex))
+    renderCodeInCanvasId(code, canvasid)
+    content = $('#' + canvasid).
+      html().
+      replace(/id=\".*?\"/g, 'id="xxx"')
+    return content
+
+  # ID counter for the equivalence tests
+  idcounter = 0
```

Comment:
Move this above `getRenderedContent` please.

Reply:
Updated, thank you.  Also makes these functions smaller, I'm using the vars from the closure.